socialdistribution/templates/posts/posts_base.html

```diff
@@ -19,20 +19,32 @@
         </div>
         <!--end of the post heading-->
         <!--start of the post item-->
+        <script>
```

Comment:
Add the js to a separate file in `static/js`.

Reply:
Change is made!